package.json

```diff
@@ -39,16 +39,16 @@
     "extends": "enact-proxy/strict"
   },
   "dependencies": {
-    "@enact/core": "^4.1.2",
-    "@enact/i18n": "^4.1.2",
-    "@enact/spotlight": "^4.1.2",
-    "@enact/ui": "^4.1.2",
+    "@enact/core": "^4.1.4",
+    "@enact/i18n": "^4.1.4",
+    "@enact/spotlight": "^4.1.4",
+    "@enact/ui": "^4.1.4",
     "classnames": "^2.3.1",
     "invariant": "^2.2.4",
     "prop-types": "^15.8.0",
     "ramda": "^0.27.1",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
     "warning": "^4.0.3",
     "ilib": "^14.11.1"
```

Comment:
```suggestion
    "ilib": "^14.14.0"
```

Reply:
"peerDependencies": {
    "ilib": "^14.14.0 || ^14.14.0-webos1"
  },
  "devDependencies": {
    "@enact/docs-utils": "^0.3.6",
    "@enact/ui-test-utils": "^1.0.0-alpha.1",
    "eslint-config-enact-proxy": "^1.0.4",
    "ilib": "^14.14.0"
  }
}

Reply:
Please update these to samples directories too.